backend/tests/export_ndjson_test.py

```
@@ -0,0 +1,446 @@
+        import time
+        time.sleep(0.1)
+
+        # Export again without overwrite
+        call_command(
+            'export_to_ndjson',
+            '--output-dir', str(self.temp_dir),
+        )
+
+        # File should not be modified
+        new_mtime = users_file.stat().st_mtime
+        self.assertEqual(original_mtime, new_mtime)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)